<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lists</class>
 <widget class="QWidget" name="Lists">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="_new_list">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="grepotool.qrc">
           <normaloff>:/icons/new.png</normaloff>:/icons/new.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="_del_list">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="grepotool.qrc">
           <normaloff>:/icons/table_delete.png</normaloff>:/icons/table_delete.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="_current_tab">
         <property name="contextMenuPolicy">
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="insertPolicy">
          <enum>QComboBox::NoInsert</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
         <property name="duplicatesEnabled">
          <bool>true</bool>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>New Item</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="_edit">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="grepotool.qrc">
           <normaloff>:/icons/edit.png</normaloff>:/icons/edit.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="_dep">
         <property name="toolTip">
          <string>Departure time</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="grepotool.qrc">
           <normaloff>:/icons/starttime.png</normaloff>:/icons/starttime.png</iconset>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="_arr">
         <property name="toolTip">
          <string>Departure time</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="grepotool.qrc">
           <normaloff>:/icons/arrivaltime.png</normaloff>:/icons/arrivaltime.png</iconset>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="_find">
         <property name="toolTip">
          <string>Departure time</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="grepotool.qrc">
           <normaloff>:/icons/town_question.ico</normaloff>:/icons/town_question.ico</iconset>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="_goto">
         <property name="toolTip">
          <string>Departure time</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="grepotool.qrc">
           <normaloff>:/icons/town_goto.ico</normaloff>:/icons/town_goto.ico</iconset>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="_info">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="_del_sel">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="grepotool.qrc">
         <normaloff>:/icons/delete.png</normaloff>:/icons/delete.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="_tab">
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>20</number>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="grepotool.qrc"/>
 </resources>
 <connections/>
</ui>
